chia/full_node/mempool_manager.py

```diff
@@ -156,6 +156,33 @@ def remove_seen(self, bundle_hash: bytes32):
         if bundle_hash in self.seen_bundle_hashes:
             self.seen_bundle_hashes.pop(bundle_hash)
 
+    def get_min_fee_increase(self):
+        # 0.00001 XCH
+        return 10000000
+
+    def can_replace(self, conflicting_items, removals, fees, fees_per_cost):
+        conflicting_fees = 0
+        conflicting_cost = 0
+        for item in conflicting_items.values():
+            conflicting_fees += item.fee
+            conflicting_cost += item.cost_result.cost
+
+            # All coins spent in all conflicting items must also be spent in
+            # the new item
+            for coin in item.removals:
+                if coin.name() not in removals:
+                    return False
+
+        # New item must have higher fee per cost
+        if fees_per_cost <= conflicting_fees / conflicting_cost:
+            return False
+
+        # New item must increase the total fee at least by a certain amount
+        if fees < conflicting_fees + self.get_min_fee_increase():
+            return False
+
+        return True
+
```

Comment:
Could we also put a log message (maybe debug or info level) for when a spenbundle is replaced? It would be good to know about that.